src/controllers/stocks/index.ts

```diff
@@ -70,6 +74,10 @@ export const stocksRouter = async function (app: FastifyInstance) {
 		},
 		async (req) => {
 			const { server_id, user_id, symbol, amount } = req.body;
+			if (!symbol || !amount || amount < 1)
```

Comment:
this line is still kind of an issue, 0 returns falsy in javascript

Reply:
Not sure I see how that is an issue? It will throw if amount is undefined, null, or less than 1, which is exactly what we want.